manifests/init.pp

```diff
@@ -36,6 +36,38 @@
   }
   validate_string($tac_plus_template_real)
 
+  if $acl == 'NONE' {
```

Comment:
since the code is still pre v1.0.0, I dont think we should put deprecation warnings in the code, since breaking changes are acceptable. Let's just document in the CHANGELOG.